pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "roiextractors"
-version = "0.6.3"
+version = "0.7.0"
```

Comment:
actually, let's drop the `write_imaging` as well whose times has come so you avoid the change of version after we remove those.

Reply:
Sounds good, can you take care of that?

Reply:
https://github.com/catalystneuro/roiextractors/pull/502